### PR TITLE
Bjorncs/https

### PR DESCRIPTION
**Pull request by bjorncs**
FYI @arnej27959 @freva

### CODE REVIEW

File: vespaclient/src/sh/vespa-get-cluster-state [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+eval $(vespa-security-env)
+
+$VESPA_HOME/libexec/vespa/GetClusterState.pl "$@"
```
Comment (vekterli) [inline vespaclient/src/sh/vespa-get-cluster-state]: Nit: missing newline

File: vespaclient/src/sh/vespa-get-node-state [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+eval $(vespa-security-env)
+
+$VESPA_HOME/libexec/vespa/GetNodeState.pl "$@"
```
Comment (vekterli) [inline vespaclient/src/sh/vespa-get-node-state]: Nit: missing newline

**Review by vekterli — APPROVED** [review]
:+1:

**Review by arnej27959 — COMMENTED** [review]
I was going to say that in theory you need a bootstrap section in each script, but you did that already.  As a minor nit it’s good to end wrapper scripts with “exec” whatever to really handover control and avoid the extra script process hanging around.

**Comment by bjorncs** [comment]
@arnej27959 Good point. Fixed here: https://github.com/vespa-engine/vespa/pull/10060.

**Pull request merged**